clingo/tests/test.bats

```diff
@@ -0,0 +1,17 @@
+@test "package directory for package ident ${TEST_PKG_IDENT} exists" {
+  [ -d "/hab/pkgs/${TEST_PKG_IDENT}" ]
+}
+
+# Refer to https://github.com/potassco/clingo/tree/master/examples/gringo/sort
+@test "gringo should work as expected for python" {
+  actual=$( hab pkg exec core/clingo gringo --text ${TESTDIR}/examples/gringo/sort/sort-py.lp ${TESTDIR}/examples/gringo/sort/encoding.lp )
```

Comment:
Tests should `hab pkg exec $TEST_PKG_IDENT gringo` to ensure the version we built is being tested

Reply:
Fixed.